swap/simulations_test.go

```diff
@@ -210,13 +213,22 @@ func newSharedBackendSwaps(t *testing.T, nodeCount int) (*swapSimulationParams,
 	// then create the single SimulatedBackend
 	gasLimit := uint64(8000000000)
 	defaultBackend := backends.NewSimulatedBackend(alloc, gasLimit)
-	testBackend := &swapTestBackend{SimulatedBackend: defaultBackend}
+	defaultBackend.Commit()
+
+	factoryAddress, _, _, _ := contractFactory.DeploySimpleSwapFactory(bind.NewKeyedTransactor(keys[0]), defaultBackend)
```

Comment:
i think you should inspect the returned `error` here, and call `t.Fail()` if different from `nil`

Reply:
done